src/pages/About/Officers.vue

```diff
@@ -0,0 +1,226 @@
+          the OCF lab and other services. They are elected by the Board of
+          Directors at the end of each semester.
+        </h4>
+        <OfficerCardSet
+          role-wanted="Site Manager"
+          :cards="cards"
+          :double-set="false"
+        />
+        <br />
+        <div class="is-divider"></div>
+        <h2 class="title is-2 has-text-centered">Committee Heads</h2>
+        <h4 class="title is-4 has-text-centered">
+          Committee heads guide the work of the various committees that function
+          within the OCF.
+        </h4>
+
+        <div class="columns is-centered">
+          <div class="column">
+            <h3 class="title is-3 has-text-centered">Internal Heads</h3>
+            <OfficerCardSet
+              role-wanted="Internal Head"
+              :cards="cards"
+              :double-set="true"
+            />
+          </div>
+          <div class="column">
+            <h3 class="title is-3 has-text-centered">
+              Industry and Alumni Relations Heads
+            </h3>
+            <OfficerCardSet
+              role-wanted="Industry and Alumni Relations Head"
+              :cards="cards"
+              :double-set="true"
+            />
+          </div>
+        </div>
+        <div class="columns is-centered">
+          <div class="column">
+            <h3 class="title is-3 has-text-centered">Finance Heads</h3>
+            <OfficerCardSet
+              role-wanted="Finance Head"
+              :cards="cards"
+              :double-set="true"
+            />
+          </div>
+          <div class="column">
+            <h3 class="title is-3 has-text-centered">Communications Heads</h3>
+            <OfficerCardSet
+              role-wanted="Communications Head"
+              :cards="cards"
+              :double-set="true"
+            />
+          </div>
+        </div>
+        <h3 class="title is-3 has-text-centered">DeCal Heads</h3>
+        <OfficerCardSet
+          role-wanted="DeCal Head"
+          :cards="cards"
+          :double-set="false"
+        />
+      </div>
+    </section>
+  </Layout>
+</template>
+
+<script>
+import OfficerCardSet from "../../components/OfficerCardSet";
+
+export default {
+  metaInfo: {
+    title: "Officers"
+  },
+  components: {
+    OfficerCardSet
+  },
+  data() {
+    return {
+      cards: []
+    };
+  },
+  created() {
+    this.cards = [
+      {
+        id: 1,
+        name: "Kevin Mo",
+        handle: "kmo",
+        role: ["General Manager"],
+        face: "https://bulma.io/images/placeholders/1280x960.png",
+        icon: "https://bulma.io/images/placeholders/96x96.png"
+      },
+      {
+        id: 2,
+        name: "Saurabh Narain ",
+        handle: "snarain",
+        role: ["General Manager"],
+        face: "https://bulma.io/images/placeholders/1280x960.png",
+        icon: "https://bulma.io/images/placeholders/96x96.png"
+      },
+      {
+        id: 3,
+        name: "Nikhil Jha  ",
```

Comment:
whitespace

Reply:
Additionally this is data that can be pulled from LDAP, but understandably without ocfapi that might be difficult